meteor/client/ui/Status/SystemStatus.tsx

```diff
@@ -182,6 +186,48 @@ export const DeviceItem = reacti18next.withTranslation()(
 			}
 		}
 
+		private getDebugStateTableBody(debugState: object) {
```

Comment:
This might be better as being its own component, to make it more reusable. It would also allow for memoizing this computation